projects/npm-tools/packages/npm-scripts/src/scripts/build.js

```diff
@@ -4,6 +4,7 @@
  */
 
 const fs = require('fs');
+const parseArgs = require('minimist');
```

Comment:
I don't know whether we'll keep doing this via an option (given the source formatter thing), but in case we do...

I think using minimist is a bit of overkill here to grab a single option.

We're [already calling `minimist()` from `src/index.js`](https://github.com/liferay/liferay-frontend-projects/blob/de5eb0773d8c19c86e1bcc4ff2a97f0eb024e5f6/projects/npm-tools/packages/npm-scripts/src/index.js#L11-L13), and now calling it again here for what would be a simple `argv.includes('--federation')`.

(Personally, I never would have added `minimist` to this project in the first place because it is using [245 lines of JS](https://github.com/substack/minimist/blob/master/index.js) to do what can be done in a single expression... 🤦‍♂️)

Reply:
The fact is that I used `minimist` because it was there :sweat_smile: . My first implementation used `includes`.

It has happened the same with the `getFixture` code: I copied my code from some other test that didn't use it.

Maybe we should refactor these things every once in a while to make sure the project looks the same in every file. I know it's not easy and it will never be perfect, but maybe we can create low priority issues for these things and do them every once in a while if we have time to spare...

Reply:
Yeah, "Boy Scout Rule" — when you see something that could be cleaned up, make it a little better along the way.